hw/lint/common.core

```diff
@@ -13,7 +13,7 @@ filesets:
   files_ascentlint:
     files:
       - common.waiver: {file_type: waiver}
-      - ascentlint.policy: {file_type: tclSource}
```

Comment:
Is this the default AscentLint policy, or has it been tailored to filter certain checks? My first impression with this tool is that it is overly pessimistic.

Reply:
This is just a generic placeholder name here, which is now being replaced with the new configuration script. 

We have always been using a tailored set of rules internally, but the policy file containing the configuration of these rules can not be open sourced. It is only now with this tool release that we can actually share the complete linting flow with partners outside of our organisation.

The rules have been derived from our current style guide. For instance, the tool would flag all `X` literals in RTL.